### PR TITLE
:sparkles: manager integration

### PR DESCRIPTION
**Pull request by yanmxa**
## Related issue(s)

Fixes #

### CODE REVIEW

File: manager/pkg/cronjob/scheduler_test.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ func TestScheduler(t *testing.T) {
 	var err error
 	testenv = &envtest.Environment{
```
Comment (clyang82) [inline manager/pkg/cronjob/scheduler_test.go:32]: maybe move it to integration

Comment (yanmxa) [reply manager/pkg/cronjob/scheduler_test.go]: Done.

**Pull request merged**